terraform/soaking/variables.tf

```diff
@@ -56,4 +56,8 @@ variable "soaking_data_rate" {
 
 variable "soaking_data_type" {
   default = "otlp"
-}
\ No newline at end of file
+}
+
+variable "data_emitter" {
```

Comment:
can you help to change the name of "data_emitter" to "sample_app_image"? in case any confusion.

Reply:
this was a variable I added that was meant to be what "sample_app" is to "sample_app_image" for data emitter images. The idea was for data_emitter to be from the imagebuild sample apps. Setting this to "sample_app_image" will overlap with a variable with a different meaning so I don't think that is the right thing to do. I think I can remove this variable and just use the variable "sample_app" because it has no other uses in ec2_setup, soaking, and performance

Reply:
On second thought I realised sample_app is in the common.tf and has a default of "spark", so it cannot be reused here, I will instead name it "soaking_sample_app"